bnbridge/src/components/instructions/instructions.jsx

```diff
@@ -118,7 +123,7 @@ class Instructions extends Component {
     })
   };
 
-  render() {
+  renderBEP2_ERC20() {
```

Comment:
breaking function naming convention: change to renderBep2ToErc20 (use lowerCamelCase)